HotelManagerment/com.spring.sample/src/com/spring/sample/entity/Room.java

```diff
@@ -2,18 +2,40 @@
 
 import java.io.Serializable;
 
-@SuppressWarnings("serial")
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+
+@Entity(name = "ROOMS")
 public class Room extends BaseEntity implements Serializable {
 
+    @Id
+    @Column(name = "RoomID")
+    @GenericGenerator(name = "assigned-generator", strategy = "assigned")
+    @GeneratedValue(generator = "assigned-generator")
     private String roomID;
```

Comment:
```suggestion
    private String id;
```
đổi tên field luôn đi, PK để id là đu